Compile with main sdk

fraidev opened the pull request:
I fixed the newtonsoft version to fix IOS build too.

Blocked:
- [x] We need the fix https://github.com/trilitech/tezos-unity-sdk/issues/77 to test the compiled build.

fraidev committed:
Compile with main sdk

fraidev committed:
Get last sdk commit

fraidev committed:
fix newtonsoft version to fix ios build

umutkutlu commented:
Tezos Unity SDK already includes the Newtonsoft.Json plugin for WebGL, [here.](https://github.com/trilitech/tezos-unity-sdk/blob/main/Runtime/Plugins/Newtonsoft.Json.dll) This is blocker for building the project to WebGL
![image](https://github.com/trilitech/tezos-unity-sdk-rpg-sample-game/assets/33513682/7f4d7874-f396-4791-b937-9d843c49ea4f)

### Workaround
**Option 1:** Use the plugin that is already included in the SDK. (Preffered)
**Option 2:** Uncheck WebGL from yours.

fraidev committed:
remove newtosoft-json

fraidev commented:
> Tezos Unity SDK already includes the Newtonsoft.Json plugin for WebGL, [here.](https://github.com/trilitech/tezos-unity-sdk/blob/main/Runtime/Plugins/Newtonsoft.Json.dll) This is blocker for building the project to WebGL ![image](https://user-images.githubusercontent.com/33513682/244878097-7f4d7874-f396-4791-b937-9d843c49ea4f.png)
> 
> ### Workaround
> **Option 1:** Use the plugin that is already included in the SDK. (Preffered) **Option 2:** Uncheck WebGL from yours.

I removed the dependency that I added before, and I created a [PR](https://github.com/trilitech/tezos-unity-sdk/issues/85) to add the plugin to IOS builds.
